frontstage/views/register.py

```diff
@@ -230,91 +220,13 @@ def register_enter_your_details():
         logger.debug("Confirmation password is: {}".format(password_confirm))
         logger.debug("phone number is: {}".format(phone_number))
 
-        # Lets try and create this user on the OAuth2 server
-        # OAuth_payload = {
-        #     "username": email_address,
-        #     "password": password,
-        #     "client_id": app.config['RAS_FRONTSTAGE_CLIENT_ID'],
-        #     "client_secret": app.config['RAS_FRONTSTAGE_CLIENT_SECRET']
-        # }
-
-        headers = {'content-type': 'application/x-www-form-urlencoded'}
```

Comment:
The headers are defined at line 280 of the new version. It was being defined twice in a row so this one wasn't used